src/main/java/org/fundacionjala/pivotal/pageobjects/workspaces/Workspaces.java

```diff
@@ -0,0 +1,56 @@
+package org.fundacionjala.pivotal.pageobjects.workspaces;
+
+import org.fundacionjala.core.ui.AbstractPage;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.fundacionjala.core.ui.AbstractPage'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)